### ISSUE TITLE
Bad results for 512 images

### ISSUE THREAD

**Issue by cuevas1208** [shepnerd/inpainting_gmcnn#13]
I followed your instructions on issue #9 by setting learning rate to 1e-5 and by changing "mask_priority = priority_loss_mask(mask)" to "mask_priority = priority_loss_mask(mask, hsize=128, sigma=1.0 / 60, iters=16)" in network.py (L178). See image bellow for my results. 
(Top left- pretrain results, Top right- fine tuning at 100 epochs, bottom left- fine tuning at 500 epochs and  Bottom right- fine tuning at 1000 epochs.)

![image](https://user-images.githubusercontent.com/13240742/56433861-463caf80-6287-11e9-81a2-55543418802f.png)

Bellow is a screenshot of my training loss. 
![image](https://user-images.githubusercontent.com/13240742/56434015-ce22b980-6287-11e9-900f-92cb6170910a.png)

am I doing anything wrong? are this results normal? #5

**Comment by shepnerd** [comment]
Did you train with random stroke masks?  When training with random stroke
masks, actually the window size (hsize) can be adjusted to ensure meaningful weights are assigned to the missing region. You can set hsize to a smaller value if the training strokes are not dense. Besides, you may tune the weight for ID-MRF loss.

Manuel Cuevas <notifications@github.com> 于2019年4月20日周六 上午12:48写道：

**Comment by cuevas1208** [comment]
Yes, I did use random strokes for training. I would look into tuning the weight for ID-MRF loss and window size (hsize). Thank you

**Comment by cuevas1208** [comment]
I was able to get rid of the saturation after tuning wgan learning rate and MRF. Thank you.

**Comment by shepnerd** [comment]
It may be caused by the channel order of the input images. The input images of the given pretrained models, test script, and the GUI tool are in BGR order, but our released train script works on RGB order. We have modified the input images in the train script into BGR images for the consistency with the pretrained model.

As your trained model takes in RGB images, you may consider re-testing the model by modifying the test script for ensuring input RGB images. In opencv, cv2.imread reads the BGR image.
